scipy/stats/stats.py

```diff
@@ -5860,7 +5860,8 @@ def _ttest_nans(a, b, axis, namedtuple_type):
 
 
 def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
-              permutations=None, random_state=None, alternative="two-sided"):
+              permutations=None, random_state=None, alternative="two-sided",
+              _comb=False):
```

Comment:
For comparing exact p-values produced using the old "permutation" method against those produced using the new "combination" method.